spec/src/main/asciidoc/opentelemetry-apis.adoc

```diff
@@ -35,12 +35,8 @@ Baggage API::
 * https://www.javadoc.io/static/io.opentelemetry/opentelemetry-api/{otel-java-version}/io/opentelemetry/api/baggage/propagation/package-summary.html[io.opentelemetry.api.baggage.propagation]
 Metrics API::
 * https://www.javadoc.io/static/io.opentelemetry/opentelemetry-api/{otel-java-version}/io/opentelemetry/api/metrics/package-summary.html[io.opentelemetry.api.metrics]
-
-
-[NOTE]
-=====
-`io.opentelemetry.api` does depend on https://www.javadoc.io/static/io.opentelemetry/opentelemetry-api/{otel-java-version}/io/opentelemetry/api/logs/package-summary.html[io.opentelemetry.api.logs], though this specification only supports the parts that relate to Tracing and Metrics.
-=====
+Logs API::
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-api/{otel-java-version}/io/opentelemetry/api/logs/package-summary.html[io.opentelemetry.api.logs]
```

Comment:
Since the logs api is intended as a bridge from existing log frameworks, and isn't meant to be a log framework, do we need to require the API to be exposed to apps?

Reply:
According to the [spec ](https://opentelemetry.io/docs/specs/otel/logs/bridge-api/), the API is not intended to be called by end users and therefore shouldn't be exposed to applications. It would be the responsibility of the runtime to export them depending on the user's configuration.